lib/docs/filters/rust/clean_html.rb

```diff
@@ -97,6 +97,22 @@ def call
           node.previous_element.before(node)
         end
 
+        css('.collapse-toggle').remove
+
+        # Fix how notable-traits sections are shown
+
+        css('.method').each do |node|
+          traitSection = node.at_css('.notable-traits')
```

Comment:
The handling of `.important-traits` in line 81 seems related. Is this related? Is `.important-traits` still in use?

Reply:
Looks like .important-traits is not in used anymore, I'm going to clean those lines